topological_navigation/scripts/topological_prediction.py

```diff
@@ -79,18 +63,21 @@ def __init__(self, name) :
         self.FremenClient.wait_for_server()
         rospy.loginfo(" ...done")
 
-        self.predict_srv=rospy.Service('/topological_prediction/predict_edges', strands_navigation_msgs.srv.PredictEdgeState, self.predict_edge_cb)
-        #This service returns a list of nodes that have a given tag
+
+        #Creating Action Server
+        rospy.loginfo("Creating action server.")
+        self._as = actionlib.SimpleActionServer(name, strands_navigation_msgs.msg.BuildTopPredictionAction, execute_cb = self.BuildCallback, auto_start = False)
```

Comment:
it's weird using the node name here, as this is the action to build a model. Why did you do it like this?